templates/workspaces/unrestricted/porter.yaml

```diff
@@ -135,6 +151,14 @@ parameters:
     type: boolean
     default: true
     description: "Setting this to `true` will prevent the need for users to manually grant consent to new workspaces"
+  - name: enable_airlock_malware_scanning
```

Comment:
There will be no airlock in the unrestricted workspace, so this should be left to the terraform defaults.